app/namespace.go

```diff
@@ -2001,6 +2019,21 @@ func disruptiveChange(old namespace.AuthMethod, new namespace.AuthMethod) bool {
 	return old != namespace.AUTH_METHOD_RESTRICTED && new != namespace.AUTH_METHOD_API_KEY_OR_MTLS
 }
 
+func getRegionId(cloudProvider, activeRegion string) (common.RegionID, error) {
+	var regionId common.RegionID
+	switch cloudProvider {
+	case CloudProviderAWS:
+		regionId.Provider = common.CLOUD_PROVIDER_AWS
+		regionId.Name = activeRegion
+	case CloudProviderGCP:
+		regionId.Provider = common.CLOUD_PROVIDER_GCP
+		regionId.Name = activeRegion
```

Comment:
As regionID.Name is both activeRegion, we could take it out of the switch clause?

Reply:
ack